converters/fromZigbee.js

```diff
@@ -5686,6 +5691,55 @@ const converters = {
             return payload;
         },
     },
+    silvercrest_smart_led_string: {
+        cluster: 'manuSpecificTuya',
+        type: ['commandGetData', 'commandSetDataResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            const dp = msg.data.dp;
```

Comment:
Don't write to `meta.state` in this converter, instead return the new state like done in other converters. e.g. 

```
const payload = {}
......
else if (dp === common.TuyaDataPoints.silvercrestSetBrightness) {
                payload.brightness = value;
} 
.....
return payload;

Reply:
Done.

Reply:
I think you missed https://github.com/Koenkk/zigbee-herdsman-converters/pull/1799/files#diff-b676ab0df4d49b8572967cb4c92a2dc6264091131f6b1828ca13d0f9656e367aR5714